Header header

geometry_msgs/Vector3 sigma_p
geometry_msgs/Vector3 sigma_v
geometry_msgs/Vector3 sigma_eul_deg
geometry_msgs/Vector3 sigma_b_a
geometry_msgs/Vector3 sigma_b_g_deg
float32 sigma_alt
geometry_msgs/Vector3 sigma_l_b_r
geometry_msgs/Vector3 sigma_eul_b_r_deg
